readthedocs/builds/models.py

```diff
@@ -258,6 +290,8 @@ def get_subdomain_url(self):
     def get_downloads(self, pretty=False):
         project = self.project
         data = {}
+        # TODO: refactor the ``pretty`` argument. It seems to just return nicer
+        # keys, but the logic looks the same
```

Comment:
Believe @stsewd already did this in another PR.

Reply:
Yes. I need to rebase my branch.